### DISCUSSION TITLE
Wait I am confused

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [vanjs-org/van#338] [Q&A] [answered]
What is the difference between vanjs and mini-van?

**Comment by Tao-VanJS** [comment, marked as answer]
* VanJS: Supports JSX-free DOM composition, states and state bindings. Suitable for CSR (client-side rendering), helpful for building feature-rich client-side apps, including SPA (single-page application).
* Mini-Van: Only supports JSX-free DOM composition (all state and state binding related APIs have only dummy implementation without reactivity). Primarily suitable for SSR (server-side rendering), though you can do CSR with Mini-Van if your app doesn't need reactivity.
* You can build shared UI components that work with both VanJS and Mini-Van, making the components reusable on both server-side and client-side.
